src/common/mockFactory.ts

```diff
@@ -268,6 +268,59 @@ export default class MockFactory {
         };
     }
 
+    /**
+     * Creates fake IV1Project
+     * @param name Name of project.
+     */
+    public static createTestV1Project(frameCount: number = 10): IV1Project {
+        const frames: { [frameName: string]: IV1Region[]; } = {};
+        for (let i = 0; i < frameCount; i++) {
+            const name = `testFrame${i}.jpg`;
+            frames[name] = MockFactory.createTestV1Regions();
+        }
+
+        return {
+            frames,
+            framerate: "1",
+            inputTags: "testTag1,testTag2",
+            suggestiontype: "suggestiontype",
+            scd: true,
+            visitedFrames: ["testFrame0.jpg"],
+            tag_colors: [MockFactory.randomColor(), MockFactory.randomColor()],
+        };
+    }
+
+    /**
+     * Creates fake IV1Region
+     */
+    public static createTestV1Regions(count: number = 3): IV1Region[] {
+        const regions: IV1Region[] = [];
+
+        for (let i = 0; i < count; i++) {
+            const testRegion = {
+                x1: 1,
```

Comment:
nit: Any way we could randomize these dimensions? Just to avoid magic numbers if possible and keep the tests honest

Reply:
There are other examples within the `MockFactory` of this kind of stuff

Reply:
Yeah I can do that. I was originally testing the entire region, but I can just test that it exists with a specific UID than testing the whole object.